src/main/java/org/takes/rq/RqForm.java

```diff
@@ -341,6 +317,36 @@ public InputStream body() throws IOException {
         }
 
         /**
+         * Validate parameters.
+         * @param params Parameters
+         * @return Validated parameters
```

Comment:
@xupyprmv Explain that the method return the parameters if their count is even, or else it throws an exception.